.gitignore

```diff
@@ -15,6 +15,9 @@ out/*
 
 # Assignments
 asn*
+## A1
+src/main/java/Country.java
+src/main/java/CountryCatalogue.java
```

Comment:
Maybe create an `assignments` package and ignore the whole directory.

Reply:
I was gonna', but there will sometimes be some specific files & classes and whatnot they'd need.

Reply:
You can have those outside of the `assignments` package.